wiki/Tournaments/MXB/3/en.md

```diff
@@ -51,6 +51,12 @@ Prizes for this tournament are sponsored by [IA daisuki](https://osu.ppy.sh/user
   - [NimaPro](https://live.bilibili.com/12166146)
   - [Rasis](https://live.bilibili.com/5333)
 
+| Placing | Player |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | [A M D](https://osu.ppy.sh/users/5321112) (a.k.a amxmodx) |
```

Comment:
If there is a name change during the tournament, i would prefer to update all references to this player more than just doing this.

Reply:
He changed his name **after** the tournament. Should I update all references?